.travis.mk

```diff
@@ -176,11 +176,11 @@ debug_ubuntu_ghactions: deps_ubuntu_ghactions test_debug_debian_no_deps
 # Coverage
 
 build_coverage_debian:
-	cmake . -DCMAKE_BUILD_TYPE=Debug -DENABLE_GCOV=ON
-	make -j $$(nproc)
+	cmake . -G Ninja -DCMAKE_BUILD_TYPE=Debug -DENABLE_GCOV=ON
+	cmake --build . --parallel $(($(nproc) + 1))
```

Comment:
Please pay attention to https://github.com/tarantool/tarantool/runs/6956800770?check_suite_focus=true#step:5:365. I can see the following:
```
cmake --build . --parallel 
```

You should use `$$` to make it work.